roles/contiv_network/tasks/main.yml

```diff
@@ -108,3 +108,6 @@
 
 - include: aci_tasks.yml
   when: (run_as == "master") and (contiv_network_mode == "aci")
+
+- include: fwd_tasks.yml
+  when: (run_as == "master") 
```

Comment:
I am not sure if we want this task to be conditional. This condition will make the task run only on `master` nodes (and we run netplugin on `worker` nodes a well). Since the flag is a netplugin flag, I think you want to run it on all nodes, right?